src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    private static final Random random = new Random();
+    private final Color[] color = Color.values();
+
+    public static String getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index].name().toLowerCase();
```

Comment:
According to the checklist, we should use `name()` for getting String representation of enum constants. However, `name()` returns the enum constant in all uppercase letters, and there is no need to convert it to lowercase unless specified by the task requirements.